site/index.html

```diff
@@ -14,7 +14,7 @@
 
     <script type="text/javascript" src="https://code.jquery.com/jquery-1.7.2.min.js"></script>
     <!-- DOMPurify is an XSS sanitiser and we definitely don't want any XSS, so best to load it -->
```

Comment:
Hmm, you're right that this is unclear. How about adding a comment like
```
<! -- TODO Some insecure browsers (Chrome!) seem to prevent us from loading this script. Figure out how to bypass their block -->

```